test/support/factories/trip_planner/trip_planner.ex

```diff
@@ -3,6 +3,7 @@ defmodule Test.Support.Factories.TripPlanner.TripPlanner do
   Provides generated test data via ExMachina and Faker.
   """
 
+  alias Test.Support.Generators
```

Comment:
Ooops. Will fix.

Although I thought we had set up Credo to catch this type of thing?